assurance/execution.test.js

```diff
@@ -0,0 +1,87 @@
+const { execute } = require('../test/helpers/execution');
+const { mapValueSets } = require('../test/helpers/valueSetMapper');
+const { loadELM, loadJSONFixture, loadValueSets } = require('../test/helpers/fixtureLoader');
+
+let valueSetMap;
+let elm;
+let patientBundle;
+
+beforeEach(() => {
+  // Set up necessary data for cql-execution
+  valueSetMap = {
+    SteamVS: {
+      1: [
+        {
+          code: 'example-code',
+          system: 'www.example.com',
+        },
+      ],
+    },
+  };
+  elm = loadJSONFixture(__dirname, 'fixtures/elm/testLib.json');
+  patientBundle = loadJSONFixture(__dirname, './fixtures/patients/test-patient-1.json');
+});
+
+test('Should properly match on resources in execution', () => {
+  const expectedCondition = patientBundle.entry[0].resource;
+  const executionResults = execute([elm], patientBundle, valueSetMap, 'testLib');
+  const patientID = '123';
+
+  expect(executionResults.patientResults[patientID].Condition).toHaveLength(1);
+  // eslint-disable-next-line no-underscore-dangle
+  expect(executionResults.patientResults[patientID].Condition[0]._json).toEqual(expectedCondition);
+});
+
+test('Should exclude resources with values outside of the valueSet Map during execution', () => {
+  // Changing the expected code within the valueSetMap to a code not included in the patient bundle
+  valueSetMap.SteamVS[1][0].code = 'second-example-code';
+  const executionResults = execute([elm], patientBundle, valueSetMap, 'testLib');
+  const patientID = '123';
+
+  // There should be no matching Condition resources within the execution results
+  expect(executionResults.patientResults[patientID].Condition).toHaveLength(0);
+});
+
+test('Should properly load patient resource from bundle', () => {
+  const executionResults = execute([elm], patientBundle, valueSetMap, 'testLib');
+  const patientID = '123';
+
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
We should definitely feel free to add this special case to our .eslintrc. Take a look at how we defined special cases for ignoring this rule in the MEF: https://github.com/mcode/mcode-extraction-framework/blob/master/.eslintrc.json

After doing that, we should remove the few disables we have throughout this file.